data/generic_nullable_vector.go

```diff
@@ -10,7 +10,17 @@ func newNullablegenVector(n int) *nullablegenVector {
 }
 
 func (v *nullablegenVector) Set(idx int, i interface{}) {
-	(*v)[idx] = i.(*gen)
+	if i == nil {
+		(*v)[idx] = nil
+		return
+	}
+	switch i.(type) {
```

Comment:
This I am less sure of. I think perhaps Set maybe shouldn't do the type switch. Perhaps a new method? This seems like the complementary method to `ConcreteAt` http://localhost:6060/pkg/github.com/grafana/grafana-plugin-sdk-go/data/#Field.ConcreteAt , so maybe ConcreteSet?

> ConcreteAt returns the concrete value at the specified index idx. A non-pointer type is returned regardless if the underlying vector is a pointer type or not. If the value is a pointer type, and is nil, then the zero value is returned and ok will be false.